packages/example/pages/_app.tsx

```diff
@@ -53,128 +53,111 @@ import {
+  zora,
+  bsc,
+  zkSync,
+  ...(process.env.NEXT_PUBLIC_ENABLE_TESTNETS === 'true'
+    ? [goerli, holesky]
+    : []),
+];
+
 const connectors = connectorsForWallets([
   ...wallets,
   {
     groupName: 'Other',
     wallets: [
-      argentWallet({ chains, projectId }),
-      bifrostWallet({ chains, projectId }),
-      bitgetWallet({ chains, projectId }),
-      bitskiWallet({ chains }),
-      clvWallet({ chains, projectId }),
-      coin98Wallet({ chains, projectId }),
-      coreWallet({ chains, projectId }),
-      dawnWallet({ chains }),
-      desigWallet({ chains }),
-      enkryptWallet({ chains }),
-      foxWallet({ chains, projectId }),
-      frameWallet({ chains }),
-      frontierWallet({ chains, projectId }),
-      imTokenWallet({ chains, projectId }),
-      ledgerWallet({ chains, projectId }),
-      mewWallet({ chains }),
-      oktoWallet({ chains, projectId }),
-      okxWallet({ chains, projectId }),
-      omniWallet({ chains, projectId }),
-      oneKeyWallet({ chains }),
-      phantomWallet({ chains }),
-      rabbyWallet({ chains }),
-      safeheronWallet({ chains }),
-      safepalWallet({ chains, projectId }),
-      subWallet({ chains, projectId }),
-      tahoWallet({ chains }),
-      talismanWallet({ chains }),
-      tokenaryWallet({ chains }),
-      tokenPocketWallet({ chains, projectId }),
-      trustWallet({ chains, projectId }),
-      uniswapWallet({ chains, projectId }),
-      xdefiWallet({ chains }),
-      zealWallet({ chains }),
-      zerionWallet({ chains, projectId }),
+      argentWallet({ projectId }),
+      bifrostWallet({ projectId }),
+      bitgetWallet({ projectId }),
+      bitskiWallet(),
+      clvWallet({ projectId }),
+      coin98Wallet({ projectId }),
+      coreWallet({ projectId }),
+      dawnWallet(),
+      desigWallet(),
+      enkryptWallet(),
+      foxWallet({ projectId }),
+      frameWallet(),
+      frontierWallet({ projectId }),
+      imTokenWallet({ projectId }),
+      ledgerWallet({ projectId }),
+      mewWallet(),
+      oktoWallet({ projectId }),
+      okxWallet({ projectId }),
+      omniWallet({ projectId }),
+      oneKeyWallet(),
+      phantomWallet(),
+      rabbyWallet(),
+      safeheronWallet(),
+      safepalWallet({ projectId }),
+      subWallet({ projectId }),
+      tahoWallet(),
+      talismanWallet(),
+      tokenPocketWallet({ projectId }),
+      tokenaryWallet(),
+      trustWallet({ projectId }),
+      uniswapWallet({ projectId }),
+      xdefiWallet(),
+      zealWallet(),
+      zerionWallet({ projectId }),
     ],
   },
 ]);
 
 const wagmiConfig = createConfig({
-  autoConnect: true,
+  chains,
   connectors,
-  publicClient,
-  webSocketPublicClient,
+  transports: {
+    [mainnet.id]: http(),
+    [polygon.id]: http(),
+    [optimism.id]: http(),
+    [arbitrum.id]: http(),
+    [base.id]: http(),
+    [zora.id]: http(),
+    [bsc.id]: http(),
+    [zkSync.id]: http(),
+    [goerli.id]: http(),
+    [holesky.id]: http(),
```

Comment:
We may need to manually add the Alchemy transports here now that `alchemyProvider` is deprecated. (like this: https://rc.wagmi.sh/react/api/transports/fallback#usage) 

At least for our example dapp and docs site. We should remove the mentions of the ALCHEMY API Key env elsewhere